docs/source/ReferenceGuide.rst

```diff
@@ -327,19 +334,22 @@ results directory structure can be seen below ::
                     ├── post
                     └── pre
     
-    21 directories, 59 files
+    22 directories, 65 files
 
 
 From what you can see, the results dir has:
 
 1) A human readable ``id`` in the top level, with the job SHA1.
 2) A human readable ``job.log`` in the top level, with human readable logs of
    the task
-3) A machine readable ``results.xml`` and ``results.json`` in the top level,
+3) Subdirectory ``jobdata``, that contains information about the job, intended
```

Comment:
contains `machine readable` data...

Also I think the second part (`intended ...`) could be dropped as the first part describes what it is and one can use it for theirs purposes.